SSA/Projects/InstCombine/LLVM/PrettyEDSL.lean

```diff
@@ -47,13 +47,12 @@ macro_rules
 
 open MLIR.AST
 
-syntax mlir_op_operand " = " "llvm.mlir.constant" neg_num (" : " mlir_type)? : mlir_op
-syntax mlir_op_operand " = " "llvm.mlir.constant" ("$" noWs "{" term "}") (" : " mlir_type)? : mlir_op
+syntax mlir_op_operand " = " "llvm.mlir.constant(" neg_num (" : " mlir_type)? ")" (" : " mlir_type) : mlir_op
+syntax mlir_op_operand " = " "llvm.mlir.constant(" ("$" noWs "{" term "}") ")" (" : " mlir_type)?   : mlir_op
```

Comment:
```suggestion
syntax mlir_op_operand " = " "llvm.mlir.constant" "(" neg_num (" : " mlir_type)? ")" (" : " mlir_type) : mlir_op
syntax mlir_op_operand " = " "llvm.mlir.constant" "(" ("$" noWs "{" term "}") ")" (" : " mlir_type)?   : mlir_op
```
I'd split this up into separate tokens, so that whitespace is allowed between them.
For example, `llvm.mlir.constant (42) : i32` would not parse with the original version